detections/endpoint/windows_find_domain_organizational_units_with_getdomainou.yml

```diff
@@ -0,0 +1,63 @@
+name: Windows Find Domain Organizational Units with GetDomainOU
+id: 0ada2f82-b7af-40cc-b1d7-1e5985afcb4e
+version: 1
+date: '2023-08-31'
+author: Gowthamaraj Rajendran, Mauricio Velazco, Splunk
+status: production
+type: TTP
+data_source:
+- Powershell 4104
+description: This analytic leverages PowerShell Script Block Logging (EventCode=4104) to detect the execution of the `Get-DomainOU` commandlet. 
+  `Get-DomainOU` is a component of PowerView, a PowerShell toolkit designed for Windows domain enumeration. 
+  Identifying the use of `Get-DomainOU` is crucial as adversaries and Red Teams might employ it to gain insights into organizational units within Active Directory, 
+  potentially aiding in lateral movement or privilege escalation strategies.
+search: '`powershell` EventCode=4104 ScriptBlockText = "*Get-DomainOU*" 
+  | stats count min(_time) as firstTime max(_time) as lastTime by EventCode ScriptBlockText Computer UserID 
```

Comment:
@gowthamarajr : lets use CIM field names for all the detections here : so please check if Computer is already available as dest when this data is indexed in splunk and User ID as user. IF they are not parsed, add a rename statement to the SPL. Also change this in the observables and message